issue-onboarding/onboarding-route.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  labels:
+    eventlistener: onboarding-issue-el
+  name: onboarding-issue-route
+spec:
+  port:
+    targetPort: http-listener
+  to:
+    kind: Service
+    name: onboarding-issue-service
```

Comment:
Remove the, aicoe-ci uses single route for its eventlistener
https://github.com/AICoE/aicoe-ci/blob/master/manifests/overlays/moc/route.yaml